AsyncDisplayKit/ASPagerFlowLayout.m

```diff
@@ -0,0 +1,63 @@
+//
+//  ASPagerFlowLayout.m
+//  AsyncDisplayKit
+//
+//  Created by Levi McCallum on 2/12/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#import "ASPagerFlowLayout.h"
+
+@interface ASPagerFlowLayout ()
+
+@property (strong, nonatomic) NSIndexPath *currentIndexPath;
+
+@end
+
+@implementation ASPagerFlowLayout
+
+- (void)invalidateLayout
+{
+  self.currentIndexPath = [self _indexPathForVisiblyCenteredItem];
```

Comment:
Because it looks like we only set this upon invalidation of the layout, we should probably call it previous page indexpath, or something else. Having trouble thinking of a good name, but I am worried that it might later be used as though it is up-to-date value at all times.

Maybe I don't totally understand all the conditions under which invalidateLayou is called, though.